janitor/functions.py

```diff
@@ -1175,9 +1175,7 @@ def remove_columns(df: pd.DataFrame, columns: List):
     :param df: A pandas DataFrame
     :param columns: The columns to remove.
     """
-    for col in columns:
-        del df[col]
-    return df
+    return df.drop(columns=columns  )
```

Comment:
@dave-frazzetto there's an extraneous space here. Please accept the suggested change.

```suggestion
    return df.drop(columns=columns)
```